doc/ref/language.xml

```diff
@@ -1129,15 +1198,19 @@ see&nbsp;<Ref Sect="Arithmetic Operations for Elements"/>.
 <Heading>Statements</Heading>
 
 <Index>execution</Index>
-Assignments (see <Ref Sect="Assignments"/>),
-Procedure calls (see <Ref Sect="Procedure Calls"/>),
-<K>if</K> statements (see <Ref Sect="If"/>),
-<K>while</K> (see <Ref Sect="While"/>),
-<K>repeat</K> (see <Ref Sect="Repeat"/>) and
-<K>for</K> loops (see <Ref Sect="For"/>),
-and the <K>return</K> statement
-(see <Ref Sect="Return (With or without Value)"/>)
-are called <E>statements</E>.
+&GAP; programs consist of a sequence of so-called  <E>statements</E>.
+The following types of statements exist:
+<List>
+  <Item>Assignments (see section&nbsp;<Ref Sect="Assignments"/>),</Item>
+  <Item>Procedure calls (see section&nbsp;<Ref Sect="Procedure Calls"/>),</Item>
+  <Item><K>if</K> statements (see section&nbsp;<Ref Sect="If"/>),</Item>
+  <Item><K>while</K> loops (see section&nbsp;<Ref Sect="While"/>),</Item>
+  <Item><K>repeat</K> loops (see section&nbsp;<Ref Sect="Repeat"/>),</Item>
+  <Item><K>for</K> loops (see section&nbsp;<Ref Sect="For"/>),</Item>
+  <Item><K>break</K> statements (see section&nbsp;<Ref Sect="Break"/>),</Item>
+  <Item><K>continue</K> statements (see section&nbsp;<Ref Sect="Continue"/>),</Item>
+  <Item><K>return</K> statements (see section&nbsp;<Ref Sect="Return"/>).</Item>
```

Comment:
In my opinion, the noun in a reference should be capitalised, e.g. "Section 1.2.3", "Chapter 4", etc. So in particular I recommend making these be `Section&nbsp;<Ref`...

I _very_ quickly searched the GAP manuals and it seems like the noun is capitalised in the majority (although by no means not all) of the cases. So perhaps the whole manual should be made consistent in this regard (although in a separate PR of course).

(And of course many of the references in the manual don't give a noun, as was the case here before your changes. I definitely prefer it when the noun is given.)

Reply:
I normally capitalize it, but when I looked around in that one XML file, I only saw the lower case variant I therefore ended up using here. But I now had another look, and I see both appear even in this file. So I am happy to change these here; but the existing inconsistent uses will remain for now.